config/clients/java/template/Java/libraries/native/apiException.mustache

```diff
@@ -0,0 +1,79 @@
+{{>licenseInfo}}
+
+package {{invokerPackage}};
+
+import java.net.http.HttpHeaders;
+
+{{>generatedAnnotation}}
+public class ApiException extends{{#useRuntimeException}} RuntimeException {{/useRuntimeException}}{{^useRuntimeException}} Exception {{/useRuntimeException}}{
```

Comment:
same, repeated

Reply:
Thanks -- Deleted in [3dbbaf7](https://github.com/openfga/sdk-generator/pull/158/commits/3dbbaf752f74a14f3535e48df92b4a934822699d)